src/content/smart-bulletins/029.md

```diff
@@ -0,0 +1,115 @@
+
+## Summary
+
+1. The interim FAR rule adopts the definition of critical technologies included in the Foreign Investment Risk Review Modernization Act of 2018 and amends sections of the FAR designed to ensure the United States is protected from certain risks regarding foreign actors.
+2. The Federal Government is prohibited from acquiring any equipment, system, or service that uses covered telecommunications equipment or services as asubstantial or essential component of any system, or as critical technology aspart of any system. This prohibition on acquiring covered telecommunications applies to acquisitions at all dollar levels including micro-purchases, unless a waiver is granted or an exception applies.
+
+## Action
+
+Agency headquarters A/OPCs shall ensure all personnel involved in purchase card use and management are aware of this new statutory prohibition on acquiring covered telecommunications. The attached Frequently Asked Questions (FAQs) developed by the Center for Charge Card Management (CCCM) is provided to assist in this effort. As noted in the FAQs:
+- To assist cardholders and card managers, a list of the sources (“vendors”) of the covered telecommunications equipment and/or services is posted on the
+GSA SmartPay® website at: https://smartpay.gsa.gov/content/prohibited-vendor-list.
+- GSA CCCM on-line purchase cardholder and A/OPC training was updated to reflect the prohibition on acquiring covered telecommunications. 
+
+Agency cardholder or card manager questions regarding the propriety of a particular telecommunications purchase should be directed to appropriate agency acquisition policy staff. Other questions or comments regarding this Smart Bulletin should be directed to CCCM through email at: gsa_smartpay@gsa.gov.
+
+## Attachment
+GSA SmartPay Section 889 FAQs
+
+## GSA SmartPay® Charge Card and Related Payment Services Program Frequently Asked Questions (FAQs) in Regard to:
+
+### Section 889 of the Fiscal Year 2019 National Defense Authorization Act and Federal Acquisition Regulation Regulation Case 2018-017, “Prohibition on Contracting for Certain Telecommunications and Video Surveillance Services or Equipment”
+
+**August 26, 2019**
+
+Background: Federal Acquisition Regulation (FAR) case 2018-017 is an interim rule implementing Section 889 of the John S. McCain National Defense Authorization Act for Fiscal Year 2019 (NDAA), which became effective on August 13, 2019. It prohibits the Federal Government from procuring or obtaining “any equipment, system, or service that uses covered telecommunications equipment or services as a substantial or essential component of any system, or as critical technology of any system.”<sup><a href="#footnote-1">1</a></sup> “Covered telecommunications or services” means: 
```

Comment:
August 26, 2019
Background: Federal Acquisition Regulation (FAR) case 2018-017 is an interim rule implementing Section 889 of the John S. McCain National Defense Authorization Act for Fiscal Year 2019 (NDAA), which became effective on August 13, 2019. It prohibits the Federal Government from procuring or obtaining “any equipment, system, or service that uses covered telecommunications equipment or services as a substantial or essential component of any system, or as critical technology of any system.”[1](https://federalist-ab31a10d-375d-4040-9324-1ae94e8a36b9.sites.pages.cloud.gov/preview/gsa/smartpay-website/feature/145-bulletins-29-to-38/policies-and-audits/smart-bulletins/029/#footnote-1) “Covered telecommunications or services” means:
Telecommunications equipment produced by Huawei Technologies Company, ZTE Corporation, or any of their subsidiaries or affiliates;
For the purpose of public safety, security of Government facilities, physical security surveillance of critical infrastructure, and other national security procedures, video surveillance and telecommunications equipment produced by Hytera Communications Corporation, Hangzhou Hikvision Digital Technology Company, or Dahua Technology Company, or any subsidiary or affiliate of such entities;
Telecommunications or video surveillance services provided by such entities or using such equipment;
Telecommunications or video surveillance equipment or services produced or provided by an entity that the Secretary of Defense, in consultation with the Director of National Intelligence or the Director of the Federal Bureau of Investigation, reasonably believes to be an entity owned or controlled by, or otherwise connected to, the government of a covered foreign country, which is defined as the People’s Republic of China.
In addition to the purchase prohibition, the FAR case requires contractors to provide a representation addressing whether or not they are providing covered telecommunications under contracts and orders with the Federal government and to report if they identify covered telecommunications equipment or services during contract performance.
FAQ 1: Are cardholders able to use micro-purchase authority to acquire covered telecommunications equipment or services?
Answer 1: No, FAR 13.201 now prohibits cardholders from procuring or obtaining any equipment, system, or service that uses covered telecommunications equipment or services as a substantial or essential component of any system or as critical technology as part of any system unless an exception applies or a waiver is granted (see the exception and waiver FAQs, below). Additionally, cardholders must adhere to the policy issued by their agency regarding prohibited purchases.
FAQ 2: The provision at FAR 52.204-24 requires contractors to provide a representation stating whether or not they are providing covered telecommunications to the Federal government. Are purchase cardholders required to obtain a representation when making a micro-purchase?
Answer 2: No, micro-purchases generally do not require provisions or clauses, and therefore, cardholders are not required to obtain the representation provision at FAR 52.204-24. However, independent of whether or not a representation is required (see FAQ 5 below), cardholders are prohibited from procuring or obtaining any equipment, system, or service that uses covered telecommunications equipment or services as a substantial or essential component of any system, or as critical technology as part of any system. Cardholders shall exercise caution to ensure such purchases are not made. (See also the exception and waiver FAQs below.)
FAQ 3: When making purchases through Federal Government buying sites such as GSA Advantage! or through Government-wide contracts or GSA/VA schedules within the micro-purchase threshold, are cardholders required to obtain the contractor representation described above in FAQ #3?
Answer 3: No, as stated in the answer to FAQ #3, because micro-purchases generally do not require provisions or clauses, cardholders are not required to obtain the representation provision at FAR 52.204-24. (But as noted in the response to FAQ 5, an agency could opt to require such representations). GSA’s Government-wide contracts, such as GSA schedules, are required to obtain representation at the contract level, which provides some level of assurance that covered telecommunications equipment and services will not be sold on that contract vehicle. Regardless of whether or not a representation is obtained, cardholders are prohibited from procuring or obtaining equipment that uses covered telecommunications equipment or services as a substantial or essential component of any system or as critical technology as part of any system and shall exercise caution to ensure that such purchases are not made.
FAQ 4: What steps should agencies take to assist cardholders in complying with this prohibition?
Answer 4: Agencies should first ensure cardholders, approving officials, and Agency/Organization Program Coordinators (A/OPCs) are aware of this prohibition, as well as any agency-specific policy pertaining to it. As noted below, the GSA SmartPay online purchase card A/OPC and cardholder training is updated to reflect the prohibition, and a list of the covered telecommunications equipment and services is posted on the GSA SmartPay website for cardholder and card manager reference. Agencies may wish to consider imposing additional precautions to ensure that purchase card transactions do not violate the prohibition. As example, agencies may require the vendor representation on any purchase that potentially includes information or communication technology or may consider imposing a temporary ban on using the GSA SmartPay purchase card to acquire telecommunications equipment or services until such time more specific information on covered items and services becomes available (specific make/model, service providers, subsidiaries, etc). Such precautions are within the agencies’ discretion.[2](https://federalist-ab31a10d-375d-4040-9324-1ae94e8a36b9.sites.pages.cloud.gov/preview/gsa/smartpay-website/feature/145-bulletins-29-to-38/policies-and-audits/smart-bulletins/029/#footnote-2)
FAQ 5: What action(s) should be taken if a vendor representation is obtained from a vendor indicating that they intend to provide a product incorporating covered telecommunications?
Answer 5: The cardholder should take action in accordance with agency policy.[3](https://federalist-ab31a10d-375d-4040-9324-1ae94e8a36b9.sites.pages.cloud.gov/preview/gsa/smartpay-website/feature/145-bulletins-29-to-38/policies-and-audits/smart-bulletins/029/#footnote-3)
FAQ 6: Do agencies that manage their own task orders under the GSA SmartPay 3 master contracts need to modify them?
Answer 6: Yes, the revisions to the FAR require existing task orders to include the reporting clause at FAR 52.204-25, “Prohibition on Contracting for Certain Telecommunications and Video Surveillance Services or Equipment,” when they are modified to extend their periods of performance, including the exercise of options. To sufficiently mitigate the risk associated with possibly procuring or obtaining covered telecommunications equipment or services, however, agencies may want to modify existing task orders as soon as practicable rather than waiting for the exercise of options.
FAQ 7: Will the GSA SmartPay online purchase cardholder and Agency/Organization Program Coordinator (A/OPC) training on the GSA SmartPay website be revised to address the prohibition?
Answer 7: Yes, the training was updated the week of August 12, 2019
FAQ 8: Is there a waiver process for acquiring covered telecommunications?
Answer 8: Yes, prohibition waiver processes are addressed in the new FAR 4.2104. Waivers may only be granted by the head (or designee) of an executive agency and only on a one-time basis, not to exceed two years. The waiver is issued for the Federal Government entity level (e.g., requirements office, contracting office) that requests it. Note that waivers must also be reported to Congress. Consult FAR 4.2104 for further information.
FAQ 9: Are there any exceptions to the prohibition?
Answer 9: Yes. The ban applies only to any equipment, system, or service that uses covered telecommunications equipment or services as a substantial or essential component of any system or as critical technology as part of any system. Further, the statute and FAR rule specifically state that agencies are not prohibited from procuring a service that connects to the facilities of a third party, such as backhaul, roaming, or interconnection arrangements. Agencies are also not prohibited from procuring telecommunications equipment that cannot route or redirect user data traffic or permit visibility into any user data or packets that such equipment transmits or otherwise handles. Please note that additional FAQ language clarifying these exceptions is under review and is expected to be added later.
FAQ 10: What if cardholders or A/OPCs have questions about whether a purchase card purchase of telecommunications equipment or services is appropriate?
Answer 10: Address such questions to appropriate agency acquisition policy officials for further guidance prior to making any purchase of concern.

1 The language provided above is summarized. The verbatim statutory procurement prohibition language is as follows: (a) Prohibition on Use or Procurement -- (1) The head of an executive agency may not — (A) procure or obtain or extend or renew a contract to procure or obtain any equipment, system, or service that uses covered telecommunications equipment or services as a substantial or essential component of any system, or as critical technology as part of any system; or (B) enter into a contract (or extend or renew a contract) with an entity that uses any equipment, system, or service that uses covered telecommunications equipment or services as a substantial or essential component of any system, or as critical technology as part of any system. (2) Nothing in paragraph (1) shall be construed to — (A) prohibit the head of an executive agency from procuring with an entity to provide a service that connects to the facilities of a third party, such as backhaul, roaming, or interconnection arrangements; or (B) cover telecommunications equipment that cannot route or redirect user data traffic or permit visibility into any user data or packets that such equipment transmits or otherwise handles.
2 Note that shifting transactions from the purchase card to other acquisition methods will have workload and other implications.
3 As an example, GSA internal policy for Contracting Officers is as follows from the GSA Acquisition Manual (GSAM) 504.7005: Post-award procedures. (a) Supply Chain Event Report. (1) If a prohibited article is discovered within the supply chain of a procurement, the contracting officer shall immediately submit a supply chain event report using the online form on the GSA Acquisition Portal (http://insite.gsa.gov/ scrm) to ensure appropriate service and staff offices within GSA are notified. (2) The supply chain event report must include the following information: (i) Contract information, including contract number and contractor name; (ii) GSA contracting office; (iii) Prohibited article name; and (iv) Reason why prohibited article is banned on contract. (3) The contracting officer shall provide as much information as is available at the time of report submission. (4) GSA’s representative to the Federal Acquisition Security Council or designee will notify the contracting officer to confirm receipt of the report.